README.md

```diff
@@ -1,2 +1,3 @@
 # S47_Weirdy_Foody
-This is your first repository
+Weirdly Foody
```

Comment:
Consider breaking down the project description into smaller, more digestible sentences or bullet points to improve readability. [medium]